testing/src/main/java/io/grpc/internal/testing/AbstractTransportTest.java

```diff
@@ -190,6 +190,21 @@ public void tearDown() throws InterruptedException {
     }
   }
 
+  /**
+   * Moves the clock foward, for tests that require moving the clock fowrard. It is the transport
+   * subclass's responsibility to implement this method.
+   */
+  protected void advanceClock(long nanos) {
```

Comment:
Include Nanos in the name, or require a TimeUnit. It was unexpected it uses different units than `currentTimeMillis()` below.

Reply:
changed to accept a timeunit